core/src/main/java/pl/agh/edu/actor/utils/LinkLabel.java

```diff
@@ -0,0 +1,85 @@
+package pl.agh.edu.actor.utils;
+
+import static pl.agh.edu.actor.utils.SkinColor.ColorLevel._300;
+import static pl.agh.edu.actor.utils.SkinColor.ColorLevel._500;
+import static pl.agh.edu.actor.utils.SkinColor.ColorLevel._700;
+import static pl.agh.edu.actor.utils.SkinColor.ColorLevel._900;
+import static pl.agh.edu.actor.utils.SkinColor.GRAY;
+import static pl.agh.edu.actor.utils.SkinColor.SECONDARY;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.InputListener;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+import com.badlogic.gdx.utils.Null;
+
+import pl.agh.edu.audio.SoundAudio;
+
+public class LinkLabel extends LanguageLabel {
+	private boolean isDisabled = false;
+	private SkinColor baseColor = SECONDARY;
+
+	public LinkLabel(String languagePath, String font, Runnable linkAction) {
```

Comment:
jeszcze widzę że jako font tu przyjmujemy stringa a trochę lepiej by było mieć właśnie FontType ale to już nie na ten PR bo to jeszcze zmianki w LanguageLabel dochodzą